pkg/cloud/services/network/subnets.go

```diff
@@ -83,6 +78,12 @@ func (s *Service) reconcileSubnets() error {
 		}
 	}
 
+	// Describe subnets in the vpc.
+	existing, err := s.describeVpcSubnets()
```

Comment:
curious what would be the difference if moving from L57 to L82 :)

Reply:
For subnets tags of the managed cluster and unmanaged cluster, I didn't see the difference and even from the testing, not sure if I understand wrong.
What's our expectations? :)

Reply:
`describeVpcSubnets` actually tags the existing subnets. But you are right, only moving this wont help.
I will take a look at subnet tagging again.

Reply:
Refer slack thread [here](https://kubernetes.slack.com/archives/CD6U2V71N/p1678252002376529).